libraries/AP_NavEKF3/AP_NavEKF3_Measurements.cpp

```diff
@@ -961,7 +961,13 @@ void NavEKF3_core::writeEulerYawAngle(float yawAngle, float yawAngleErr, uint32_
 
     yawAngDataNew.yawAng = yawAngle;
     yawAngDataNew.yawAngErr = yawAngleErr;
-    yawAngDataNew.type = type;
+    if (type == 2) {
```

Comment:
As an alternative to this - should we push the `rotationOrder` enum out to callers to `writeEulerYaw`?

Maybe later - esp. given @rmackay9 's comment on using AP's normal enumeration list...

Reply:
I would like to do that as a separate PR.